service/debugger/debugger.go

```diff
@@ -1401,15 +1362,15 @@ func (d *Debugger) Ancestors(goroutineID, numAncestors, depth int) ([]api.Ancest
 			r[i].Unreadable = fmt.Sprintf("could not read ancestor stacktrace: %v", err)
 			continue
 		}
-		r[i].Stack, err = d.convertStacktrace(frames, nil)
+		r[i].Stack, err = d.ConvertStacktrace(frames, nil)
 		if err != nil {
 			r[i].Unreadable = fmt.Sprintf("could not read ancestor stacktrace: %v", err)
 		}
 	}
 	return r, nil
 }
 
-func (d *Debugger) convertStacktrace(rawlocs []proc.Stackframe, cfg *proc.LoadConfig) ([]api.Stackframe, error) {
+func (d *Debugger) ConvertStacktrace(rawlocs []proc.Stackframe, cfg *proc.LoadConfig) ([]api.Stackframe, error) {
```

Comment:
I think we should move this off of `Debugger` and into `api` as it's own function. Same goes for any other `Convert*` methods on `Debugger`.

Can happen in another commit though.